balancer-js/src/modules/data/index.ts

```diff
@@ -133,6 +131,7 @@ export class Data implements BalancerDataRepositories {
         url: networkConfig.urls.subgraph,
         chainId: networkConfig.chainId,
         blockHeight: blockDayAgo,
+        query: subgraphQuery,
```

Comment:
Not sure if this is actually intended, but I feel like if the user pass a subgraphQuery, they should be applied to all repositories. Let me know if that is not correct.

Reply:
Yeah, that makes sense to me and should be much more consistent to follow what the effects are.